plugins/push/frontend/public/templates/mobile-message-preview.html

```diff
@@ -48,10 +84,10 @@
     </div>
 
     <div class="cly-vue-push-notification-mobile-preview__platforms">
-        <div v-if="hasIOSPlatform" @click="setSelectedPlatform(PlatformEnum.IOS)" v-bind:class="['cly-vue-push-notification-mobile-preview__icon-container',isIOSPlatformSelected?'cly-vue-push-notification-mobile-preview__active-icon':'']">
+        <div v-if="hasIOSPlatform()" @click="setSelectedPlatform(PlatformEnum.IOS)" v-bind:class="['cly-vue-push-notification-mobile-preview__icon-container',isIOSPlatformSelected?'cly-vue-push-notification-mobile-preview__active-icon':'']">
             <span class="ion-social-apple"></span>
         </div>
-        <div v-if="hasAndroidPlatform" @click="setSelectedPlatform(PlatformEnum.ANDROID)" v-bind:class="['cly-vue-push-notification-mobile-preview__icon-container',isAndroidPlatformSelected?'cly-vue-push-notification-mobile-preview__active-icon':'']">
+        <div v-if="hasAndroidPlatform()" @click="setSelectedPlatform(PlatformEnum.ANDROID)" v-bind:class="['cly-vue-push-notification-mobile-preview__icon-container',isAndroidPlatformSelected?'cly-vue-push-notification-mobile-preview__active-icon':'']">
```

Comment:
computed here again?

Reply:
We had this discussion on one of the previous prs and I ended up breaking the component when I switched them to computed. The reason is because findInitialSelectedPlatform() method is used upon component state initialization and from a watcher. findINitialSelectedPlatform() method uses hasAndroidPlatform() and hasIOSPlatform() methods, so if I convert them to computed then it will not work since computed->data and not the opposite. Same thing from the watcher, since I am changing the state there too.
I can see that this is quite confusing so perhaps I can try to change the methods to computed and introduce a method findPlatform(platform)?

Reply:
Sorry I just thought that it went unnoticed. It is fine if you know what you're doing.